### PR TITLE
Delete child

### PR DESCRIPTION
**Pull request by R1SsophieFEHU**
Method delete child add on Family component

### CODE REVIEW

File: front/src/components/Family.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import { connect } from 'react-redux'
+import Axios from 'axios';
```
Comment (ghost) [inline front/src/components/Family.jsx]: axios is not a component, so camelCase will fit better than PascalCase

File: front/src/components/Family.jsx [hunk 2]
```diff
@@ -65,7 +67,7 @@ class Family extends React.Component {
       method: 'post',
       headers: {
         'Content-Type': 'application/json',
-        'Authorization': this.props.token
+        'Autorization': this.props.token
```
Comment (rocambille) [inline front/src/components/Family.jsx]: are you sure?

https://developer.mozilla.org/fr/docs/Web/HTTP/Headers/Authorization

**Review by rocambille — CHANGES_REQUESTED** [review]

File: front/src/components/Family.jsx [hunk 3]
```diff
@@ -82,6 +84,22 @@ class Family extends React.Component {
     }
   }
 
+  deleteChild = (id) => {
+    Axios(`http://localhost:8000/children/${id}`, {method:'delete'})
```
Comment (ghost) [inline front/src/components/Family.jsx]: same about axios here

**Review by ghost — CHANGES_REQUESTED** [review]

File: front/src/components/Family.jsx (hunk 3 above)
Comment (R1SsophieFEHU) [reply front/src/components/Family.jsx]: modified and push

File: front/src/components/Family.jsx (hunk 2 above)
Comment (R1SsophieFEHU) [reply front/src/components/Family.jsx]: Modified Authorization and not Autorization...

**Pull request merged**